imgui.go

```diff
@@ -1222,3 +1227,8 @@ func MouseCursor() int {
 func SetMouseCursor(cursor int) {
 	C.iggSetMouseCursor(C.int(cursor))
 }
+
+// SetKeyboardFocusHere gives keyboard focus to next item
+func SetKeyboardFocusHere(offset int) {
```

Comment:
`offset` is a default parameter. Please add make this function to be `SetKeyboardFocusHereV` and add a second function `SetKeyboardFocus()` which calls the verbose one with `0` as argument.